src/components/Button/StrokeButton.test.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import React from 'react';
+import { mount } from 'enzyme';
+import StrokeButton, { Background } from './StrokeButton';
+
+describe('StrokeButton component', () => {
+  it('should render button with stroke outline', () => {
+    const wrapper = mount(<StrokeButton />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render background with active style', () => {
+    const colors = StrokeButton.defaultProps.strokeColors;
```

Comment:
Is this line required? These are the defaults so I don't think we need to set them explicitly. Am I missing something?

Reply:
I'm testing the styled-component `Background` here, so there is no default. Maybe I could change this to test `StrokeButton` with active style. I'll have a look later today.